### PR TITLE
RELEASE 2023-06-01

### PR DESCRIPTION
**Pull request by leite08**
### Description

- https://github.com/metriport/metriport/pull/409
- https://github.com/metriport/metriport/pull/401
- https://github.com/metriport/metriport/pull/411

### Release Plan

- ⚠️ Pointing to `master`
- Deployed packages 
- nothing special, just a deploy to production

### CODE REVIEW

File: api/app/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@aws-sdk/client-cloudwatch": "^3.338.0",
     "@medplum/core": "^2.0.18",
     "@medplum/fhirtypes": "^2.0.18",
-    "@metriport/api": "^3.1.6",
+    "@metriport/api": "^4.0.2-alpha.0",
```
Comment (Goncharo) [inline api/app/package.json:29]: do we need to update this to be the proper release version?

File: packages/packages/api/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/api",
-  "version": "4.0.1",
+  "version": "4.0.2-alpha.0",
```
Comment (Goncharo) [inline packages/packages/api/package.json:3]: ditto

File: api/app/package.json (hunk 1 above)
Comment (Orta21) [reply api/app/package.json:29]: Oh yea we will need to deploy the proper one i can do that

**Review by Goncharo — APPROVED** [review]
:shipit:

**Review by Orta21 — APPROVED** [review]

**Pull request merged**